docs/release-notes/wksp-slot-caps-webui.rst

```diff
@@ -0,0 +1,37 @@
+:orphan:
+
+**New Features**
+
+-  WebUI: A new section called "Namespace Bindings" has been added to the Create Workspace and Edit
+   Workspace modals. For OSS users, they can input a namespace that they want to bind their
+   workspace to while using a Kubernetes RM. This will allow them to send all their workloads for
+   that workspace to that particular namespace. If they do not specify a namespace, the workspace
+   will be bound to the namespace specified in the ``resource_manager.default_namespace`` field in
+   the master config YAML file. If this field is left blank, then Determined will use the
+   ``default`` namespace instead. For EE users, an additional option of auto-creating a namespace,
+   and setting the resource quota for that namespace has been added. Users can only set resource
+   quotas in the WebUI for namespaces created by Determined. The Edit Workspace Modal will display
+   the enforced resource quota for all namespaces bound to the workspace.
```

Comment:
WebUI: 
- A new section called "Namespace Bindings" has been added to the Create Workspace and Edit Workspace modals. OSS users can input a namespace to which they want to bind their workspace for a given Kubernetes cluster. If no namespace is specified, the workspace will be bound to the namespace specified in the ``resource_manager.default_namespace`` field in the master configuration YAML. If this field is left blank, then the workspace will be bound to the default Kubernetes ``default`` namespace instead. 
- MLDE users have the additional option of auto-creating a namespace to which the workspace will be bound. If this option is selected, users can also set a resource quota on that auto-created namespace. This will limit the GPU requests available to that workspace from a given Kubernetes cluster. The Edit Workspace Modal will display the enforced resource quota placed on the workspace, which is the lowest GPU limit resource quota that exists within the bound Kubernetes namespace.
- Once the workspace-namespace binding is saved, all workloads created in that workspace will be sent to the bound namespace. If a user decides to change their workspace-namespace binding, future workloads will get sent to the new namespace, but old workloads that are still in progress will remain running.